README.md

```diff
@@ -123,34 +123,39 @@ variable if it is not already configured using
 
 ### Installing HarbourBridge
 
-Download the tool to your machine and install it.
+You can make a copy of the HarbourBridge codebase from the github repository
+and use "go run":
 
 ```sh
-GO111MODULE=on go get github.com/cloudspannerecosystem/harbourbridge
+git clone https://github.com/cloudspannerecosystem/harbourbridge
+cd harbourbridge
+pg_dump mydb | go run github.com/cloudspannerecosystem/harbourbridge -driver=pg_dump
```

Comment:
we should probably skip pg_dump here and just let them run go run with the flags. The subsequent db specific sections will cover this piping use cases.

Reply:
Done.